Dockerfile

```diff
@@ -1,27 +1,19 @@
 # syntax=docker/dockerfile:experimental
-FROM ubuntu:20.04 as baseline
+FROM ubuntu:22.04 as baseline
+ENV DEBIAN_FRONTEND=noninteractive
 RUN apt update && apt upgrade -y \
   && apt install -y python3 python3-dev python3-pip curl unzip \
   && update-alternatives --install /usr/bin/python python /usr/bin/python3 1 \
   && update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 
-FROM baseline as tool_builder
-ARG kustomize_version=3.7.0
-ARG kubectl_version=1.22.10
-
-WORKDIR /build
-
-RUN curl -sLO https://storage.googleapis.com/kubernetes-release/release/v{$kubectl_version}/bin/linux/amd64/kubectl && chmod 755 ./kubectl \
-  && curl -sLO https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize/v${kustomize_version}/kustomize_v${kustomize_version}_linux_amd64.tar.gz && gunzip -c ./kustomize_v${kustomize_version}_linux_amd64.tar.gz | tar xvf - && chmod 755 ./kustomize
-
 # Installation
 FROM baseline
 ARG HELM_VERSION=3.8.1
 ARG aws_cli_version=2.1.20
 ARG gcp_cli_version=334.0.0
```

Comment:
As a note for myself, with the change to ubuntu:22.04 for the base image, python 3.10 is the default.
The gcp_cli_version=334.0.0 (3/30/21) is incompatible with it. https://issuetracker.google.com/issues/205238176

The latest gcloud cli should have this resolved. Since we use this CLI for a couple of postgres tasks will need to include a test scenario for an external PG DB in GCP.